examples/acm-terraform-blog-part3/README.md

```diff
@@ -0,0 +1,88 @@
+# Enable ACM features with Terraform - Part 3
+
+This is Part3 of the tutorial to accompany a short series of  blog articles explaining how to enable [Anthos Config Management (ACM)](https://cloud.google.com/anthos/config-management) with Terraform.
+
+In the [first part](../acm-terraform-blog-part1), we explained how to use Teraform to create a cluster and manage its config from git via [Config Sync](https://cloud.google.com/anthos-config-management/docs/config-sync-overview).
+
+In the [second part](../acm-terraform-blog-part2) we added guard rails for the cluster configuration via [Policy Controller](https://cloud.google.com/anthos-config-management/docs/concepts/policy-controller).
+
+In this article we'll demonstrate how, using Config Connector, you can provision your GCP cloud resources following the same GitOps K8s native model.
+
+## Provision GCP resources
+
+1. Set the variable for the project from [part2](../acm-terraform-blog-part2). We will re-use that project but create a new cluster since we cleaned up at the end of the first section. If you are working in a different project, enable required GCP APIs, as described in [part1/README.md](../part1/README.md).
+
+    ```bash
+    PROJECT_ID = [PROJECT_ID]
+    ```
+1. Note that [wordpress-bundle.yaml](./config-root/wordpress-bundle) was updated to use GCP MySQL database. Also we added [configconnector.yaml](./config-root/configconnector.yaml) to initialize the instance of Config Connector add-on on the cluster.
+
+1. Use [kpt](kpt.dev) to customize the `config-root` directory that will be configured as the source of the objects installed on the cluster.
+
+    ```bash
+    kpt fn eval --image gcr.io/kpt-fn/apply-setters:v0.1 ./config-root -- projectId=$PROJECT_ID
```

Comment:
We now have a fancy new "set-project-id" function which can set the project ID automatically without having to use setters: https://catalog.kpt.dev/set-project-id/v0.1/

Please switch to that, as we're trying to discourage using setters for these generic use cases.

Reply:
@AlexBulankou Looks like this is still unchanged?

Reply:
The main reason is simplicity. To avoid duplicating apply-setters logic or calling one kpt function from another. Other functions could generate new resources that also need the setter to be applied. And it was written with assumption that `kpt fn render` will always be called before apply.

Reply:
I think we should do both—apply the value directly, but also add it to setters.

The `apply-setters` logic should be modularized so it can be used elsewhere as well.

Reply:
I made the change to use `set-project` and `kpt fn render`